diff --git a/node_modules/@angular/router/fesm5/router.js b/node_modules/@angular/router/fesm5/router.js
index 436f912..3822314 100755
--- a/node_modules/@angular/router/fesm5/router.js
+++ b/node_modules/@angular/router/fesm5/router.js
@@ -7,7 +7,7 @@
 import { __values, __decorate, __param, __metadata, __extends, __assign, __spread } from 'tslib';
 import { Component, ɵisObservable, ɵisPromise, NgModuleRef, InjectionToken, NgModuleFactory, NgZone, isDevMode, ɵConsole, Attribute, Directive, ElementRef, HostBinding, HostListener, Input, Renderer2, ChangeDetectorRef, ContentChildren, QueryList, ComponentFactoryResolver, EventEmitter, Output, ViewContainerRef, Compiler, Injectable, Injector, NgModuleFactoryLoader, ANALYZE_FOR_ENTRY_COMPONENTS, APP_BOOTSTRAP_LISTENER, APP_INITIALIZER, ApplicationRef, Inject, NgModule, NgProbeToken, Optional, SkipSelf, SystemJsNgModuleLoader, Version } from '@angular/core';
 import { from, of, EmptyError, Observable, BehaviorSubject, Subject } from 'rxjs';
-import { concatAll, every, last, map, mergeAll, catchError, first, mergeMap, concatMap, reduce, filter } from 'rxjs/operators';
+import { concatAll, every, last, map, mergeAll, catchError, first, mergeMap, concatMap, reduce, filter, delay } from 'rxjs/operators';
 import { LocationStrategy, APP_BASE_HREF, HashLocationStrategy, LOCATION_INITIALIZED, Location, PathLocationStrategy, PlatformLocation, ViewportScroller } from '@angular/common';
 import { ɵgetDOM } from '@angular/platform-browser';
 
@@ -1414,7 +1414,7 @@ var ApplyRedirects = /** @class */ (function () {
                 }
                 throw e;
             }));
-        }), concatAll(), first(function (s) { return !!s; }), catchError(function (e, _) {
+        }), concatAll(), delay(100), first(function (s) { return !!s; }), catchError(function (e, _) {
             if (e instanceof EmptyError || e.name === 'EmptyError') {
                 if (_this.noLeftoversInUrl(segmentGroup, segments, outlet)) {
                     return of(new UrlSegmentGroup([], {}));
@@ -1428,9 +1428,6 @@ var ApplyRedirects = /** @class */ (function () {
         return segments.length === 0 && !segmentGroup.children[outlet];
     };
     ApplyRedirects.prototype.expandSegmentAgainstRoute = function (ngModule, segmentGroup, routes, route, paths, outlet, allowRedirects) {
-        if (getOutlet(route) !== outlet) {
-            return noMatch(segmentGroup);
-        }
         if (route.redirectTo === undefined) {
             return this.matchSegmentAgainstRoute(ngModule, segmentGroup, route, paths);
         }
